src/index.js

```diff
@@ -1,59 +1,41 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import * as serviceWorker from './serviceWorker';
-import ApolloClient from 'apollo-boost';
-import { gql } from "apollo-boost";
 import { ApolloProvider } from '@apollo/react-hooks';
 import { BrowserRouter, Route } from 'react-router-dom';
+import { createBrowserHistory } from "history";
 
 import 'bootstrap/dist/css/bootstrap.min.css';
-import SignUp from './signupPage';
-import SignIn from './signInPage';
+import SignUp from './pages/signupPage'; //Si hay una carpeta ya de una toma el idex.js de esa carpeta
+import SignIn from './pages/signInPage';
+import Home from './pages/home';
+import NavbarComp from "./components/navbar.js"
 
-const client = new ApolloClient({
-  uri: 'http://localhost:3001/graphql',
-});
+import client from './apollo/configurations/client' 
 
+const history = createBrowserHistory();
 
-
-/*
-client
-  .query({
-    query: gql`
-      {
-        user(id: "2c5ea4c0-4067-11e9-8bad-9b1deb4d3b7d"){
-          firstName
-          lastName
-          username
-        }
-      }
-    `
-  })
-  .then(result => console.log(result)); // No me esta devolviendo el result, pero la query la hace 
-
-console.log(client)
-*/
+const view = (Page) => (
+  <div><NavbarComp></NavbarComp>{Page}</div>
+);
```

Comment:
I like this approach of creating a HOC to optionally append a Navbar to other components, but let's try to do it like this:

```suggestion
const withNavbar = (component) => (
  <>
    <NavbarComp />
    {component}
  </>
)

// Then use it really similar to what you were doing on Routes
<Route exact path='/signin'>
  {withNavbar(<SignIn />)}
</Route>
```

Let's discuss a few concepts used here on Slack.